src/helpers/utils.js

```diff
@@ -97,20 +97,22 @@ export default class Utils {
 
   ensureFileDirectoryExistence(filePath) {
     var dirname = path.dirname(filePath)
-    this.ensureDirectoryExistence(dirname)
+    return this.ensureDirectoryExistence(dirname)
   }
 
 
   ensureDirectoryExistence(dirname) {
-    console.log(dirname)
-    if (fs.existsSync(dirname)) {
-      return true
+    if (!fs.existsSync(dirname)) {
+      this.mkDirRecursive(dirname)
```

Comment:
If you happened to want to bump up the minimum Node version to 10 ;) we could just do this 
```suggestion
      fs.mkdirSync(dirname, { recursive: true})
```

which would also make `mkDirRecursive` unnecessary

Reply:
Will check at some point if that's realistic :) If so, we'll do that, cause that recursive param looks way nicer